### PR TITLE
GPII-4289: For docked QS, calculating the height rather than using the window size.

### PR DESCRIPTION
**Pull request by stegru**
The new more panel increases the QS window rect, which ended up being included in the desktop area reserved for the QS.

Works with https://github.com/GPII/gpii-app/pull/176

### CODE REVIEW

**Comment by amb26** [comment]
Failure to restore Windows magnifier - this shit happens sometimes
````
07:53:00.113:  jq: Expected: {
    "gpii.windows.enableRegisteredAT": {
        "com.microsoft.windows.magnifier": [
            {
                "settings": {
                    "running": true
                }
            }
        ],
        "com.microsoft.windows.onscreenKeyboard": [
            {
                "settings": {
                    "running": false
                }
            }
        ]
    }
}
07:53:00.113:  jq: Actual: {
    "gpii.windows.enableRegisteredAT": {
        "com.microsoft.windows.magnifier": [
            {
                "settings": {
                    "running": undefined
                }
            }
        ],
        "com.microsoft.windows.onscreenKeyboard": [
            {
                "settings": {
                    "running": false
                }
            }
        ]
    }
}
````

**Comment by amb26** [comment]
ok to test

File: gpii/node_modules/gpii-windowManagement/src/appBar.js [hunk 1]
```diff
@@ -292,24 +294,26 @@ gpii.windows.appBar.disable = function (that) {
  * Gets the position of the taskbar.
  *
  * @param {Rectangle} rect [optional] The bounding rectangle of the taskbar.
- * @return {String} "left", "top", "right", or "bottom".
+ * @return {String} "left", "top", "right", "bottom", or undefined if there's no taskbar.
```
Comment (javihernandez) [inline gpii/node_modules/gpii-windowManagement/src/appBar.js:297]: I'm curious, aside the use case where the windows taskbar is hidden, are there any other cases where there is no taskbar?

Comment (stegru) [reply gpii/node_modules/gpii-windowManagement/src/appBar.js:297]: When the language is being changed, and the taskbar process is being restarted.

Also, the computer can also be configured to not start explorer at login.

**Comment by javihernandez** [comment]
I tested this as part of https://github.com/GPII/gpii-app/pull/176 and is working as expected.
LGTM :+1:

**Pull request merged**